src/citrine/resources/table_config.py

```diff
@@ -283,7 +282,7 @@ def get(self, uid: Union[UUID, str], version: Optional[int] = None):
             path = self._get_path(uid)
             data = self.session.get_resource(path)
             version_numbers = [version_data['version_number'] for version_data in data['versions']]
-            index = np.argmax(version_numbers)
+            index = version_numbers.index(max(version_numbers))
```

Comment:
I am assuming `version_numbers` is a list of unique integers here

Reply:
yes, that's correct